locales/en/LC_MESSAGES/language_live_page.po

```diff
@@ -10,16 +10,16 @@ msgid "sidebar_title"
 msgstr "Live CD language"
 
 msgid "desc_language"
-msgstr "Language:"
+msgstr "Language :"
```

Comment:
Nope, in English there's no space before the punctuation like `:`, `!`, `?`,  etc.

There is one in French, though!